### DISCUSSION TITLE
how do i add custom component into demo block

### DISCUSSION THREAD

**Discussion by racktime** [vuepress-theme-hope/vuepress-theme-hope#1115] [Q&A] [answered]
custom component display none into demo block 
how do i add custom component into demo block ?

**Comment by Mister-Hope** [comment, marked as answer]
The docs is providing everything you need. I am not getting your point. So I will assume you are not a native English speaker and you are using translator.

If you do need help, at least providing what you want, what you wrote, what's happening and what are you expecting.

**Comment by racktime** [comment]
[radio.md](https://github.com/vuepress-theme-hope/vuepress-theme-hope/files/7427994/radio.md
[cepo-ui-docs.zip](https://github.com/vuepress-theme-hope/vuepress-theme-hope/files/7427997/cepo-ui-docs.zip)
) 
thank u hope
I'm trying to write a custom component sample in vuepress next
I'm trying to demo a sample vue componet that referenced the cepo-ui.ssr.js library.

**Reply by Mister-Hope** [reply]
Could you provide some screenshots about the render result and console errors?
 Just some reminder here:

- You may need to config isCustomComponent in vueOptions
- Vue do not support `style scoped` in runtime, so I am using shadown dom, that results  `document` is not the real document. If your components is using `document`, it may not working.

**Comment by racktime** [comment]
cepo-ui-docs.zip > source repo

cepo-ui-docs\docs\.vuepress\clientAppEnhance.js

CepoUiDev  lib import use

import CepoUiDev from '../../libs/cepo-ui-dev.ssr';


//import './css/index.css'

export default ({ app, router }) => {
    app.use(CepoUiDev)
  }

**Comment by racktime** [comment]
CodeDemoOptions  jsLib: string[];

is jsLib an option for thirdparty lib setting ?
How do I set it up?
